test/test_networking.py

```diff
@@ -310,7 +318,7 @@ def setup_class(cls):
 
 
 class TestHTTPRequestHandler(TestRequestHandlerBase):
-    @pytest.mark.parametrize('handler', ['Urllib', 'Requests'], indirect=True)
+    @pytest.mark.parametrize('handler', ['Urllib', 'Requests', 'CurlCFFI'], indirect=True)
```

Comment:
Why not put `['Urllib', 'Requests', 'CurlCFFI']` in a var? Or maybe even the whole decorator:

```py
def run_for_handlers(handlers=('Urllib', 'Requests', 'CurlCFFI')):
    return pytest.mark.parametrize('handler', handlers, indirect=True)
```

Reply:
It may not always be the case, not all of these tests run for curl cffi. There are some minor changes I'll make in another PR that allows us to do this